be/src/storage/tablet_updates.cpp

```diff
@@ -1302,6 +1302,8 @@ void TabletUpdates::_apply_normal_rowset_commit(const EditVersionInfo& version_i
         for (auto& delvec_pair : new_del_vecs) {
             tsid.segment_id = delvec_pair.first;
             manager->set_cached_del_vec(tsid, delvec_pair.second);
+            // try to set empty dcg cache, for improving latency when reading
+            manager->set_cached_empty_delta_column_group(_tablet.data_dir()->get_meta(), tsid);
```

Comment:
Why not need to capture the status of this function?

Reply:
Because if we capture that we have to set "error status" to this tablet, which will make this tablet unusable. Setting empty delta column group is just improvement, not necessary.